cpp/ps0004/main.cpp

```diff
@@ -4,24 +4,26 @@ using namespace std;
 
 int main()
 {
-    int num  ;
-    char character  ;
-    cout << "Enter a number from 32 to 126 :  \n" ;
-    cin >> num  ;
-
-    if ( (float)num )
-        cout <<"Invalid: a floating point number \n";
-    else if (  num <32 || num > 126  )
-        cout <<"Invalid: number is out of the range! \n " ;
-    else  if (  num = character )
-        cout <<"Invalid character/s or zero \n " ;
-
-
+    // prompting user for number 1/1.5
+    float num;
+    char character;
+    cout << "Enter a number from 32 to 126 :  \n";
+    cin >> num;

```

Comment:
- [x] declared a variable to store the user input
- [ ] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input